### PR TITLE
Increased the min ETH balance for a validator to become active to over 32.25 ETH

### PR DESCRIPTION
**Pull request by naddison36**
## Change

Due to hysteresis, if the effective balance is 31 ETH, the actual balance must rise to 32.25 ETH to trigger an effective balance update to 32 ETH. https://eth2book.info/capella/part2/incentives/balances/#hysteresis

## Code Change Checklist

To be completed before internal review begins:

- [ ]  The contract code is complete
- [ ]  Executable deployment file
- [ ]  Fork tests that test after the deployment file runs
- [ ]  Unit tests *if needed
- [ ]  The owner has done a [full checklist review](https://github.com/OriginProtocol/security/blob/master/templates/Contract-Code-Review.md) of the code + tests

Internal review:

- [ ] Two approvals by internal reviewers

## Deploy checklist

Two reviewers complete the following checklist:

```
- [ ] All deployed contracts are listed in the deploy PR's description
- [ ] Deployed contract's verified code (and all dependencies) match the code in master
- [ ] Contract constructors have correct arguments
- [ ] The transactions that interacted with the newly deployed contract match the deploy script.
- [ ] Governance proposal matches the deploy script
- [ ] Smoke tests pass after fork test execution of the governance proposal
```

### CODE REVIEW

File: contracts/test/strategies/compoundingSSVStaking.js [hunk 1]
```diff
@@ -2725,6 +2724,86 @@ describe("Unit test: Compounding SSV Staking Strategy", function () {
...
+
+      await compoundingStakingSSVStrategy.snapBalances();
+
+      // Set validator balance to 32.26 Gwei
+      await mockBeaconProof.setValidatorBalance(
+        testValidator.index,
+        parseUnits("32.26", 9)
+      );
+
+      const tx = await compoundingStakingSSVStrategy.verifyBalances(
```
Comment (sparrowDom) [inline contracts/test/strategies/compoundingSSVStaking.js:2789]: good tests

**Review by sparrowDom — APPROVED** [review]
LGTM. what an edge case

**Review by sparrowDom — APPROVED** [review]
